### ISSUE TITLE
MS Teams alert card does not show info on mobile interface

### ISSUE THREAD

**Issue by jpds** [prometheus/alertmanager#3943]
**What did you do?**

Configured Alertmanager to send alerts to a chat in Teams with the Workflows webhook. I'm also using the patch at: https://github.com/prometheus/alertmanager/pull/3921

**What did you expect to see?**

An alert description sent to the monitoring chat, this is fine on desktops, but on mobiles - currently on the alert title is displayed.

**What did you see instead? Under which circumstances?**

Just the alert title, there's no other alert information included which makes it hard for mobile users to action alerts.

**Environment**

* System information: NixOS 24.05

* Alertmanager version: 0.27.0.

* Prometheus version: 2.53.1

**Comment by grobinson-grafana** [comment]
Hello! 👋 Does this happen on official Alertmanager releases, or just custom builds using #3921? If the latter, please close this issue and let's bring the discussion to the PR instead.

**Comment by jpds** [comment]
It's on both.

**Comment by grobinson-grafana** [comment]
On the official Alertmanager release, how are you using Power Automate workflows? Are you using one of the workarounds suggested in #3920? Can you also test with Office365 connectors to see if it's still broken?

**Comment by jpds** [comment]
On the official release - I was using the old webhooks functionality - that is broken in the same way (as with the PR code + Power Automate).

**Comment by grobinson-grafana** [comment]
But it works on desktop right?

**Comment by grobinson-grafana** [comment]
But it works on desktop? Could you share a screenshot of the same message on both desktop and mobile?

**Comment by jpds** [comment]
Of course, here's what it looks like on a desktop with all the extra info in place:

![alertmanager-teams-card-desktop](https://github.com/user-attachments/assets/6fefd8ca-9f6a-4100-a415-598869c6281d)

And here it is on the mobile app:

![alertmanager-teams-card-mobile](https://github.com/user-attachments/assets/165892b4-9d19-4f01-bb10-8be3f56393a4)

There's also nothing else available on a mobile when it is pressed/held down.

**Comment by grobinson-grafana** [comment]
I think this is a bug in Microsoft Teams app, not Alertmanager.

**Comment by fabriciodf** [comment]
Could you please try to change the version of your card? Try version 1.4

**Comment by jpds** [comment]
@fabriciodf Thanks for the suggestion, I've adjusted the value at:

- https://github.com/prometheus/alertmanager/pull/3921/files#diff-0e06d07cf4972aad8c889cb25bdb906e6b7901d2839a932ad4fccc7e60005490R154

Will see what the next alert shows in the 📱 app.

**Comment by jpds** [comment]
Sadly no change. @zhan9san would you happen to have any ideas about this?

**Comment by zhan9san** [comment]
@jpds 

Thank you for reaching out to me.

If I understand correctly, both Alertmanager and Power Automate are responsible for sending the request with the appropriate payload. If the message does not display the information as expected, I don't believe the issue lies with Alertmanager.

**Comment by khanhnt99** [comment]
> Sadly no change. @zhan9san would you happen to have any ideas about this?

Hello. Do you fix this issue?

**Comment by zhan9san** [comment]
@khanhnt99 

I am afraid I have answered the question in my previous reply. https://github.com/prometheus/alertmanager/issues/3943#issuecomment-2277586928

Do you still have any concerns?

**Comment by khanhnt99** [comment]
> @fabriciodf Thanks for the suggestion, I've adjusted the value at:
> 
> * https://github.com/prometheus/alertmanager/pull/3921/files#diff-0e06d07cf4972aad8c889cb25bdb906e6b7901d2839a932ad4fccc7e60005490R154
> 
> Will see what the next alert shows in the 📱 app.

Do you fix this issue?

**Comment by grobinson-grafana** [comment]
Thanks! I will close this issue if there are no other reports of this being broken in 0.28.1.

**Comment by jpds** [comment]
Can confirm that this is fixed in `0.28.1`.
